apps/frontend/src/pages/project/libs/helpers/check-has-project-permission.helper.ts

```diff
@@ -0,0 +1,23 @@
+import { type PermissionKey } from "~/libs/enums/enums.js";
+import { type ValueOf } from "~/libs/types/types.js";
+import { type UserAuthResponseDto } from "~/modules/auth/auth.js";
+
+const checkHasProjectPermission = (
+	authenticatedUser: null | UserAuthResponseDto,
```

Comment:
we need to pass only permissions here, not the whole authenticated user, the same as we do for checkHasPermissions. maybe for this it will be more convenient to store permissions in the slice as an object where key is project id and value is permission array